genesyscloud/util/feature_toggles/guides.go

```diff
@@ -0,0 +1,15 @@
+package feature_toggles
+
+import "os"
+
+const guideEnvToggle = "ENABLE_GUIDE_RESOURCE"
+
+func GuideToggleName() string {
+	return guideEnvToggle
+}
+
+func GuideToggleExists() bool {
+	var exists bool
+	_, exists = os.LookupEnv(guideEnvToggle)
+	return exists
+}
```

Comment:
you might have misunderstood what I was saying about checking the toggle. I meant make an API request to [GET /api/v2/guides](https://developer.genesys.cloud/platform/preview-apis#get-api-v2-guides) to check if the feature toggle was enabled in GrandCentral. If the request returns 5xx error, skip the tests. I don't think there's any need for an env var feature toggle